src/components/event/SimpleModal/Simple-modal.js

```diff
@@ -1,68 +1,61 @@
-import React, {Component} from "react";
+import React, { useState} from "react";
 import DialogActions from "@material-ui/core/DialogActions";
 import Button from "@material-ui/core/Button";
 import Dialog from "@material-ui/core/Dialog";
-import {DialogContent} from '@material-ui/core';
+import { DialogContent } from "@material-ui/core";
```

Comment:
Please move this import statement into top of file. It's because all imports from external packages needs to be on top